data/actionsmap/yunohost.yml

```diff
@@ -291,6 +291,17 @@ domain:
                     extra:
                         pattern: *pattern_domain
 
+        ### domain_generate_dns_config()
+        generate_dns_configuration:
```

Comment:
As for the API route, it's quite a long and not really nice cli command - e.g. it will be `yunohost domain generate_dns_configuration`... Note that it's possible to use -, so what about *dns-conf* or something like that?